src/test/java/pages/LinkedinPage.java

```diff
@@ -0,0 +1,50 @@
+package pages;
+
+import core.BaseClass;
+import org.junit.Assert;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.junit.Assert'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://engci-sonar-sjc.cisco.com/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)